src/Microsoft.Build.Utilities.ProjectCreation/Package.cs

```diff
@@ -193,7 +193,7 @@ internal void AddContentFile(ManifestContentFiles manifestContentFiles)
         /// <param name="versionRange">The <see cref="VersionRange" /> of the dependency.</param>
         /// <param name="include">The <see cref="LibraryIncludeFlags" /> of the assets to include.</param>
         /// <param name="exclude">The <see cref="LibraryIncludeFlags" /> of the assets to exclude.</param>
-        /// <param name="suppressParent">The <see cref="LibraryIncludeFlags" /> of the assets to suppress from dependents.  The default is <see cref="LibraryIncludeFlags.Analyzers" /> | <see cref="LibraryIncludeFlags.Build "/> | <see cref="LibraryIncludeFlags.ContentFiles" />.</param>
+        /// <param name="suppressParent">The <see cref="LibraryIncludeFlags" /> of the assets to suppress from dependents.  The default is <see cref="LibraryIncludeFlags.Analyzers" /> | <see cref="LibraryIncludeFlags.Build" /> | <see cref="LibraryIncludeFlags.ContentFiles" />.</param>
```

Comment:
This concept here is completely new to me - does this have to do with transitive Dependencies?

Reply:
ahh so if i were to add a analyzer package to my nuget package as dependency nuget doesnt carry them over - and by removing that flag i enable nuget to transitively add the analyzer package using the analyzer include?

Reply:
That's how I understand it yes